src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+    <link rel="stylesheet" href="./style.css">
+    <script src="https://kit.fontawesome.com/9b54de88d2.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
```

Comment:
I suggest that you add at least an **h1 Element** as a web page without a header violets principles of **Accessibility** and **Semantics**. certain nature of users will have difficulty.